src/msg/Message.h

```diff
@@ -558,5 +558,9 @@ template<class T, typename... Args>
 MURef<T> make_message(Args&&... args) {
   return {new T(std::forward<Args>(args)...), &TOPNSPC::common::unique_ptr_release};
 }
+template<class T>
+MURef<T> make_null_message() {
```

Comment:
Oops, it's introduced after the first usage (speaking of the order of commits).

Reply:
will fix the ordering